packages/sentry-client/src/utils/prettyTime.ts

```diff
@@ -0,0 +1,42 @@
+export function pretty(seconds: number, scale: number = 1): string {
```

Comment:
Using a library for this might be better. Dealing with the oddities once you get to months becomes very complicated very quickly